### PR TITLE
Don't reload configuration when rancher server is down

### PR DESCRIPTION
**Pull request by wacken89**
### What does this PR do?
Doesn't reload Traefik configuration when Rancher sever is down

### Motivation
We using Traefik in production, it critical for us if Rancher server is down and traefik does not proxying  requests

### CODE REVIEW

**Comment by ldez** [comment]
Happy to see your first Go PR 🎉 

Could you fix those issues:

```
---> Making bundle: validate-golint (in .)
Errors from golint:
provider/rancher/api.go:95:8: var checkRancherApi should be checkRancherAPI
provider/rancher/api.go:95:25: var rancherErrApi should be rancherErrAPI
```

File: provider/rancher/api.go [hunk 1]
```diff
@@ -92,18 +92,26 @@ func (p *Provider) apiProvide(configurationChan chan<- types.ConfigMessage, pool
...
-
-							configuration := p.buildConfiguration(rancherData)
-							if configuration != nil {
-								configurationChan <- types.ConfigMessage{
-									ProviderName:  "rancher",
-									Configuration: configuration,
+							checkRancherApi, rancherErrApi := rancherClient.ApiKey.List(withoutPagination)
+
+							if rancherErrApi != nil {
+								log.Errorf("Cannot establish connection: %+v, rancher API return: %+v;\nSkipping refresh Data from Provider API", rancherErrApi, checkRancherApi)
```
Comment (ldez) [inline provider/rancher/api.go]: could you remove the `\n` and add a `.` at the end of the sentence.

Comment (wacken89) [reply provider/rancher/api.go]: Done

**Comment by ldez** [comment]
@wacken89 Could you rebase ? (The CI failed due to a fixed pb with one test.)

**Pull request merged**